MDSassignment/guestbook/post/models.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+from django.utils import timezone
+
+
+STATUS_CHOICES = (
+	('h', 'Hidden'),
```

Comment:
Check out the diff here to see what I mean:

https://github.com/sesh/mds-assignment-ab-d/commit/43e4f8abac16895f3852997fcb38fe2d242ce880